requirements_test.txt

```diff
@@ -7,5 +7,6 @@ pytest-cov~=4.0
 pytest-profiling~=1.7
 pytest-xdist~=3.2
 # Type packages for mypy
+six
```

Comment:
We already had tests depending on `six`, and we were already installing six as a transitive dependency of `pytest-profiling`, but I figure it's better to make it explicit in case that ever changes.